README.md

```diff
@@ -12,7 +12,9 @@ It is forked from the [Cumulus](https://github.com/paritytech/cumulus) repositor
 
 Build sudo-patched polkadot binary
 ```bash
+cd ..
```

Comment:
Remove this plz, don't see what good that does

Reply:
i use it to indicate that the polkadot repo should not be cloned inside the encointer-parachain repo but in the same parent directory

Reply:
Well, then that's what you should explain IMO. Your instructions don't start inside a particular path, so the cd.. Doesn't strike me as meaningful

Reply:
get it. when reading a readme of a repo i usually assume that the commands should be executed from the repo root. i will make this more clear